staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -254,13 +254,18 @@ type PersistentVolumeClaimInfo struct {
 	// +optional
 	VolumeMode *k8sv1.PersistentVolumeMode `json:"volumeMode,omitempty"`
 
-	// Capacity represents the capacity set on the corresponding PVC spec
+	// Capacity represents the capacity set on the corresponding PVC spec ("host" capacity)
 	// +optional
 	Capacity k8sv1.ResourceList `json:"capacity,omitempty"`
 
 	// Preallocated indicates if the PVC's storage is preallocated or not
 	// +optional
 	Preallocated bool `json:"preallocated,omitempty"`
+
+	// PossibleGuestSize is the most capacity we can offer the guest, taking
+	// into account file system overhead
+	// +optional
+	PossibleGuestSize *int64 `json:"possibleGuestSize,omitempty"`
```

Comment:
I mention this in another comment as well.

What if we kind of reversed what we're storing on the VMI's status to use a term that's super concrete like `hostFilesystemOverhead`, `fileSystemOverhead`, or maybe simply `Overhead`. Which is documented to mean the amount of the Capacity used for the host filesystem that the disk.img resides on.